core/genesis.go

```diff
@@ -279,6 +288,78 @@ type ChainOverrides struct {
 	ApplySuperchainUpgrades  bool
 }
 
+// apply applies the chain overrides on the supplied chain config.
+func (o *ChainOverrides) apply(cfg *params.ChainConfig) (*params.ChainConfig, error) {
```

Comment:
Note that all our overrides were moved into this new method, which was introduced upstream